src/Sticky/Sticky.doc.js

```diff
@@ -53,15 +58,15 @@ card(
           <Text>This should stick</Text>
         </Box>
       </Sticky>
-      <Box marginTop={10}>
+      <Box marginTop={10} position="relative">
```

Comment:
updated the docs - first example is the 90% case where you shouldn't need to set zindex manually.  second example is one where you would